.travis/install.sh

```diff
@@ -44,5 +44,5 @@ CMAKE_OPTIONS="$CMAKE_OPTIONS -DGSL_ROOT_DIR=${GSL_ROOT_DIR}"
 CMAKE_OPTIONS="$CMAKE_OPTIONS -DCMAKE_CXX_FLAGS_DEBUG=${CMAKE_CXX_FLAGS_DEBUG} -O1"
 CMAKE_OPTIONS="$CMAKE_OPTIONS -DCMAKE_BUILD_TYPE=RelWithDebInfo"
 cmake .. ${CMAKE_OPTIONS} || fail "cmake failed"
-make all -j2 VERBOSE=1 || fail "make failed"
+make all -j VERBOSE=1 || fail "make failed"
```

Comment:
This is currently not failing, but it can be dangerous in some situations with too many files. `-j` means "compile as many files in parallel as possible" rather than "compile as many files are there are CPUs on this machine"; in other words `-j` is unconstrained, so if your project had more files you can run out of memory, and the OS will kill your compilation, leading to a build failure.

I'd *suggest* (but up to you) sticking with the old `-j2` because in Travis that's how many CPUs you are given (I think). You can alternatively count the number of CPUs (e.g., `grep processor /proc/cpuinfo  | wc -l`) and use that.

Reply:
Only removed as I found some builds that appeared to be timing out in a later branch when running linting after tests. I'll explore this change more in that PR.